tests/e2e/scale/test_scale_pvc_expand.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import pytest
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    E2ETest, scale, ignore_leftovers, skipif_ocs_version
+)
+from ocs_ci.ocs.scale_lib import FioPodScale
+from ocs_ci.utility import utils
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='session')
+def resize_pvc(request):
```

Comment:
There seems to be already a function with the same name `resize_pvc` in pvc.py. do you want to use a different name here?

Reply:
I'm calling the same function resize_pvc() in pvc.py thus name should be the same.